yarn-project/simulator/src/private/unconstrained_execution_oracle.ts

```diff
@@ -5,49 +5,47 @@ import type { AuthWitness } from '@aztec/stdlib/auth-witness';
 import { AztecAddress } from '@aztec/stdlib/aztec-address';
 import type { CompleteAddress, ContractInstance } from '@aztec/stdlib/contract';
 import { siloNullifier } from '@aztec/stdlib/hash';
-import type { AztecNode } from '@aztec/stdlib/interfaces/client';
 import type { KeyValidationRequest } from '@aztec/stdlib/kernel';
 import { IndexedTaggingSecret, LogWithTxData } from '@aztec/stdlib/logs';
 import type { NoteStatus } from '@aztec/stdlib/note';
 import { type MerkleTreeId, type NullifierMembershipWitness, PublicDataWitness } from '@aztec/stdlib/trees';
 import type { BlockHeader, Capsule } from '@aztec/stdlib/tx';
 
 import { type NoteData, TypedOracle } from './acvm/index.js';
-import type { DBOracle } from './db_oracle.js';
+import type { ExecutionDataProvider } from './execution_data_provider.js';
 import { pickNotes } from './pick_notes.js';
 
 /**
  * The execution context for a client view tx simulation.
```

Comment:
Could you update this comment please?

Reply:
Done. The old comment was also a lie since some things do get updated even if running unconstrained (you can do `deliverNote`)